### ISSUE TITLE
add favicon

### ISSUE THREAD

**Issue by jarmo** [jarmo/hackersays#10]
favicon is missing.

**Comment by jarmo** [comment]
This really critical functionality is still missing. @shoobomb do you have some nice examples for consideration?

**Comment by shoobomb** [comment]
Yep, Jarmo we can add the favicon. Let me know if there is some logo for Hackersays or i would have to figure out some different favicon.

**Comment by jarmo** [comment]
We do not have any logo yet and the idea is that we could add here some concepts of the logo/favicon so we can choose the best one.

**Comment by shoobomb** [comment]
Okey !!

**Comment by shoobomb** [comment]
See The "Hacker Says" banner on the top left side of the front page of the website can be an option for the favicon.

**Comment by jarmo** [comment]
We could try, but it could be too detailed/small to actually work as a favicon.

**Comment by shoobomb** [comment]
![favicon](https://cloud.githubusercontent.com/assets/8604458/12543850/cef710fc-c359-11e5-8f17-ead466869dd2.jpg)
how about this as our favicon

**Comment by moll** [comment]
Or how about a quote character? Quotes are commonly associated with... quotes. :)

**Comment by shoobomb** [comment]
yep we can. Nyc Idea!
